### ISSUE TITLE
Text review: Moderation decision email seems to assume content was innapproriate?

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#336]
@ilmartyrk @anettlinno is it correct that this same email is sent to all who have reported something regardless of what the decision of the moderator was? As the email says, "You can see the decision below" which suggests that the moderator could have either decided the report was unfounded and do nothing, or that the report was founded and should be hidden. But then the "what happens next?" text is assuming that the decision was in agreement with the report and that the content was inappropriate? I think we need two different emails to send out depending on the moderators decision, and worded differently accordingly? (Probably not just for this one, but for others too - would need a review). 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/1ded75ae-369b-4d6d-8af7-c675650b81e7)

**Comment by anettlinno** [comment]
@BeccaMelhuish you are right! There should be two different emails: 

1. When topic has been reported, our system will send out notification to topic admin(s) and moderator(s) about reported topic. "A moderator has received the report about your Citizen OS topic..." and then topic information and What happens next
2. E-mail with the decision that moderators made

**Comment by BeccaMelhuish** [comment]
@anettlinno Ahh yes I see now - I think the issue here is just that the 'What happens next?' box shouldn't be here. We already have other emails (as you described in point 1) that include this, so it can be removed from this one. 

Will leave removing that for @ilmartyrk though as it's more complex :)

**Comment by BeccaMelhuish** [comment]
@anettlinno @ilmartyrk I saw there is now still this bit, which initially I was thinking seems to be being sent regardless of the decision made, but assuming that the decision was to hide it (as per my initial issue raised above).  

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/166270d9-ca2a-4ffa-bbdc-0ff15c66d879)

However, in going through all these emails I'm understanding more how the moderation process works, and seeing that it's quite confusing and there's somewhat of a big flaw in it! 

If I'm not mistaken, I'm now guessing that this email actually only gets sent when the decision is to hide the topic. As from what I understood here [in this issue](https://github.com/citizenos/citizenos-fe/issues/443), the system is currently set up so that "Mark as resolved" is being used to mean "Disagree with the reporter and don't hide it" (as well as mark it as resolved after it has been edited - so conflating two actions under one action, which then only makes sense for one of them).

So in which case, whenever a moderator disagrees with a report and doesn't hide something, our system presumably sends out one of the "The issue you reported has now been resolved" emails. Which obviously won't make much sense, if they go to the topic/content and see it's exactly the same as it was when they reported it! (As it wasn't resolved, it was just disagreed with). 

So basically, I think there's a bigger issue with the moderation system here, which needs to be resolved before these emails can be, and will be connected to whatever flow @kevincrepin designs/has designed to solve [this issue](https://github.com/citizenos/citizenos-fe/issues/443). 

And I think we need to do some focused user testing of the moderation system at some point (but not how it currently works due to the above issues - we should work out something that works first :))

**Comment by BeccaMelhuish** [comment]
Same applies for this email, maybe one other too, would need to check: 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/8767b205-0cc1-4a88-8a78-f36c65a14ae3)

**Comment by ilmartyrk** [comment]
I am marking it as phase II
